<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCanvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>도구(&amp;T)</string>
    </property>
    <widget class="QMenu" name="menu_P">
     <property name="title">
      <string>펜(&amp;P)</string>
     </property>
     <addaction name="action_PenRed"/>
     <addaction name="action_PenGreen"/>
     <addaction name="action_PenBlue"/>
    </widget>
    <addaction name="menu_P"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>도움말(&amp;H)</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_T"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_PenRed"/>
   <addaction name="action_PenGreen"/>
   <addaction name="action_PenBlue"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/new.png</normaloff>:/icon/imgs/new.png</iconset>
   </property>
   <property name="text">
    <string>새 그림(N)</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/open.png</normaloff>:/icon/imgs/open.png</iconset>
   </property>
   <property name="text">
    <string>열기(O)</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/save.png</normaloff>:/icon/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>저장(&amp;S)</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/exit.png</normaloff>:/icon/imgs/exit.png</iconset>
   </property>
   <property name="text">
    <string>종료(X)</string>
   </property>
  </action>
  <action name="action_PenRed">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/red.png</normaloff>:/icon/imgs/red.png</iconset>
   </property>
   <property name="text">
    <string>RED(&amp;R)</string>
   </property>
  </action>
  <action name="action_PenGreen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/green.png</normaloff>:/icon/imgs/green.png</iconset>
   </property>
   <property name="text">
    <string>Green(&amp;G)</string>
   </property>
  </action>
  <action name="action_PenBlue">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/blue.png</normaloff>:/icon/imgs/blue.png</iconset>
   </property>
   <property name="text">
    <string>BLUE(&amp;B)</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/about.png</normaloff>:/icon/imgs/about.png</iconset>
   </property>
   <property name="text">
    <string>이 프로그램은...(&amp;A)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
